### ISSUE TITLE
Game crashes during specific effects or lighting 

### ISSUE THREAD

**Issue by K-Rug** [Vita3K/Vita3K-Android#475]
In toukiden 2 the game crashes when some monsters do a move that has a specific effect or lighting I'm not sure which is it. Same thing in mortal Kombat during johnny cage's intro the game crashes due to the green effect. I'd appreciate it if this was looked into. My phone specs are Mediatek helio g88,4 gb ram/126 rom mali g52 mc2.

**Comment by Jonny0779** [comment]
Да, у меня тоже самое из за зелёного эффекта у Джонни у меня вылетает.
Хотя на других персонажах не вылетает.
Когда враг или я делаю сальто за Джонни Кейджа игра вылетает.

**Comment by Levirince** [comment]
Toukiden 2 always crashes during bladetail mission in my case

**Comment by InahiSB** [comment]
> Toukiden 2 always crashes during bladetail mission in my case

You can disable your NGS support at Settings→Emulation. It makes Toukiden 2 run very smooth and will not crash your game when fighting Bladetail but at a cost of no audio/in game sounds. But don't worry, you can Enable NGS support again after Bladetail 🙂👍.
